src/main/webapp/styles.css

```diff
@@ -0,0 +1,23 @@
+.flex-container {
```

Comment:
Nit: I think you need to add the header https://github.com/google/new-project#source-code-headers

Reply:
Sounds good, thanks!

Reply:
`RESOLVED`